test/test_cache_stores.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+
+import pytest
+import responses
+
+from adr.util.cache_stores import SeededFileStore
+
+here = Path(__file__).resolve().parent
+
+
+@pytest.fixture
+def archive_response():
+    archive_dir = here / "files" / "cache_archives"
+
+    def callback(resp):
+        resp.callback_processd = True
+
+        name = resp.url.split('/')[-1]
+        fh = open(archive_dir / name, 'rb')
+        resp.raw = fh
+
+        return resp
+
+    with responses.RequestsMock(response_callback=callback) as rsps:
```

Comment:
Nit: you can use assert_all_requests_are_fired=True to make sure all mock responses you add are fired

Reply:
According to the docs, that is the default behaviour (and is inline with my experience running the tests as well).